Parse/Internal/PFDevice.m

```diff
@@ -27,14 +27,19 @@
     const char *charName = [name UTF8String];
 
     size_t size;
-    sysctlbyname(charName, NULL, &size, NULL, 0);
+    if (sysctlbyname(charName, NULL, &size, NULL, 0) < 0) {
```

Comment:
I think not, because sysctlbyname always returns `-1` on failure or `0` on success (as `man 3 sysctl` says)